File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.py
import sys
input=lambda:sys.stdin.readline().rstrip()
idx=0
while True:
    n0=int(input())
    if n0==0: break
    idx+=1
    n1=3*n0
    n2=(n1 if n1%2==0 else n1+1)//2
    n3=3*n2
    n4=n3//9
    print(f"{idx}. {'odd' if n1%2!=0 else 'even'} {n4}")

File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.c
#include <stdbool.h>
#include <stdio.h>
int main(void) {
    int n0, n1, n2, n3, n4, idx = 0;
    while (true) {
        scanf("%d", &n0);
        if (n0 == 0) break;
        idx += 1;
        n1 = 3 * n0;
        n2 = (n1 % 2 == 0 ? n1 : n1 + 1) / 2;
        n3 = 3 * n2;
        n4 = n3 / 9;
        printf("%d. %s%d\n", idx, n1 % 2 != 0 ? "odd " : "even ", n4);
    }
    return 0;
}